### PR TITLE
Mf 295 health check web

### PR DESCRIPTION
**Pull request by ChughAkshay**

### CODE REVIEW

File: src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        public AccountApiHealthCheck(
+            IAccountApiClient apiClient,
+            ILogger<AccountApiHealthCheck> logger)
+        {
+            _apiClient = apiClient;
+            _logger = logger;
+        }
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = new CancellationToken())
+        {
+            _logger.LogInformation("Pinging Commitments API");
```
Comment (dashton82) [inline src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs]:
```suggestion
            _logger.LogInformation("Pinging Accounts API");
```

File: src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+            try
+            {
+                var timer = Stopwatch.StartNew();
+                await _apiClient.GetStatistics();
```
Comment (dashton82) [inline src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs]: Can we call GetAccount(1) instead? It doesnt matter if it returns a 404, as long as its not a 500 response code

File: src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs (hunk 1 above)
Comment (scottywakefield) [inline src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs]: I'm guessing this is Account API

Comment (ChughAkshay) [reply src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs]: updated

File: src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs (hunk 2 above)
Comment (ChughAkshay) [reply src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs]: Updated

**Pull request merged**